### DISCUSSION TITLE
Associate quick tag bundles with specific blogs

### DISCUSSION THREAD

**Discussion by wizardtoads** [AprilSylph/XKit-Rewritten#1176] [Suggestion Box]
I'd love it if I could associated my tag bundles with specific blogs, and have the quick tag list be tailored to where I am reblogging to. 

i.e, When reblogging a post to a fandom blog, I see my fandom tags. When reblogging something to photography related blog, I see my photography tags. 

I've seen this come up a few times in relation to categories or something similar, but I haven't seen it suggested as blog specific- hopefully I didn't miss a suggestion. I think this would help with the big scrolling list of tag bundles, and may be simpler UI wise than categories.

I just switched to xkit rewritten from new xkit, and the overhaul is amazing. Quick Reblog and Quick Tag make Tumblr for me. Ty keep up the good work!!

**Comment by martienne17** [comment]
I second this. Relatedly, I'd love it if the auto-tagging option were also blog-specific. I use the tag #replied for asks in my main but #asks on most of my other blogs.
